### PR TITLE
enable multiple values for capsule-user-group

### LINKED ISSUE #133: Add support for multiple capsuleUserGroup
As we introduced multiple `capsuleUserGroups` in coming Capsule Operator v0.1.0, we should reflect this enhancement also in Capsule Proxy.

### PR DESCRIPTION
**Pull request by alegrey91**
Close #133

### CODE REVIEW

**Comment by alegrey91** [comment]
**TODO**
* fix helm chart files
* adjust `return` in `CheckUserInCapsuleGroupMiddleware` function
* rebase with **issue/130** branch features

File: charts/capsule-proxy/README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Parameter | Description | Default
 `options.listeningPort` | Set the listening port of the capsule-proxy.| `9001`
 `options.logLevel` | Set the log verbosity of the capsule-proxy with a value from 1 to 10.| `4`
 `options.k8sControlPlaneUrl` | Set the URL of kubernetes control plane. | `https://kubernetes.default.svc`
-`options.capsuleUserGroup` | Override the Capsule user group | `capsule.clastix.io`
+`options.capsuleUserGroups` | Override the Capsule user group | `capsule.clastix.io`
```
Comment (prometherion) [inline charts/capsule-proxy/README.md:70]: The default value should be a slice of string: `[ "capsule.clastix.io" ]`

**Review by prometherion — CHANGES_REQUESTED** [review]
Just need to fix the docs.

CI is broken, we can merge it and I'll fix it on my own.

File: charts/capsule-proxy/README.md (hunk 1 above)
Comment (alegrey91) [reply charts/capsule-proxy/README.md]: Done!

**Review by prometherion — CHANGES_REQUESTED** [review]
Could you squash the following commits 21441e040d320e26cdfd00fe6bf7327867bfaff6 21441e040d320e26cdfd00fe6bf7327867bfaff6 21441e040d320e26cdfd00fe6bf7327867bfaff6 21441e040d320e26cdfd00fe6bf7327867bfaff6 in a single one named `feat: support for multiple Capsule groups`?

**Pull request merged**